packages/dam-app-base/src/Editor/SortableComponent.tsx

```diff
@@ -128,7 +204,14 @@ const SortableList = SortableContainer<SortableContainerProps>((props: SortableC
   const { list } = props.resources.reduce(
     (acc, resource, index) => {
       const [url, alt] = props.makeThumbnail(resource, props.config);
-      const item = { url, alt, key: `url-unknown-${index}` };
+
+      let additionalData = null;
+
+      if (props.getAdditionalData) {
+        additionalData = props.getAdditionalData(resource);
+      }
```

Comment:
I feel this can be simplified to:
```suggestion
      const additionalData = props.getAdditionalData?.(resource) || null;
```